docker-centos/Dockerfile

```diff
@@ -16,6 +16,10 @@ rpm -V docker-rhel-push-plugin && \
 mkdir -p /exports/hostfs/etc/docker && \
 yum clean all
 
+# Move the /etc/sysconfig/docker out of the way as daemon.json is the proper config location
```

Comment:
@giuseppe / @rhatdan would it be easier for upgrading to drop ``/etc/sysconfig/docker`` from ``service`` template?

The problem is if both ``daemon.json`` and ``/etc/sysconfig/docker`` are used, docker fails to start. It notes that it received the same options (even if the values are the exact same).